beacon-chain/execution/engine_client.go

```diff
@@ -537,22 +537,31 @@ func handleRPCError(err error) error {
 	}
 	switch e.ErrorCode() {
 	case -32700:
+		errParseCount.Inc()
```

Comment:
one more error, if the error return by the execution client doesn't have the correct error type.